lib/spoom/snapshot.rb

```diff
@@ -0,0 +1,105 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Spoom
+  module Coverage
+    extend T::Sig
+
+    sig { params(path: String).returns(Snapshot) }
+    def self.snapshot(path: '.')
+      snapshot = Snapshot.new
+      metrics = Spoom::Sorbet.srb_metrics(path: path, capture_err: true)
+      return snapshot unless metrics
+
+      snapshot.files = metrics["types.input.files"] || 0
```

Comment:
Super nitpicky but I think `fetch` is preferred in these cases:
```suggestion
      snapshot.files = metrics.fetch("types.input.files", 0)
```